### PR TITLE
fill the unfilled row with blank elements.

### PR DESCRIPTION
**Pull request by LavaCxx**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### Description
The scroll bar affects the width of the icons, so I fill the unfilled rows with blank elements to keep the width of the icons consistent with the other rows.
image

### What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Mobile & Desktop Screenshots/Recordings
**before**
**↓↓↓↓↓**
before

**after**
**↓↓↓↓↓**
after

### Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [x] 🙅 no documentation needed

### CODE REVIEW

File: src/ts/component/menu/smile.tsx [hunk 1]
```diff
@@ -24,6 +24,11 @@ const LIMIT_SEARCH = 12;
 const HEIGHT_SECTION = 40;
 const HEIGHT_ITEM = 40;
 const ID_RECENT = 'recent';
+const ID_BLANK = 'blank';
+
+const BLANK_ITEM = {
+	itemId:ID_BLANK
```
Comment (ra3orblade) [inline src/ts/component/menu/smile.tsx:30]: Fix formatting, space after colon

File: src/ts/component/menu/smile.tsx [hunk 2]
```diff
@@ -86,7 +91,7 @@ class MenuSmile extends React.Component<I.Menu, State> {
 				const Item = (item: any) => {
 					const str = `:${item.itemId}::skin-tone-${item.skin}:`;
 					return (
-						<div 
+						item.itemId === ID_BLANK ? <div className="item"></div> : <div
```
Comment (ra3orblade) [inline src/ts/component/menu/smile.tsx:94]: both are strings, no need for strict equality, ==, use self-closing div, all empty elements should be self-closing

File: src/ts/component/menu/smile.tsx [hunk 3]
```diff
@@ -360,6 +365,12 @@ class MenuSmile extends React.Component<I.Menu, State> {
 			return (section.id == ID_RECENT) ? res : res + section.children.length; 
 		}, 0);
 
+		const fillRowWithBlank = (rowChildren) => {
+			if(rowChildren.length > 0 && rowChildren.length < LIMIT_ROW) {
```
Comment (ra3orblade) [inline src/ts/component/menu/smile.tsx:369]: Add space after if, add parenthesises for condition statements (rowChildren.length > 0), move rowChildren.length to a variable

**Comment by ra3orblade** [comment]
And another comment - there is no check in click that element is blank, so there will be an error, I assume

**Review by ra3orblade — APPROVED** [review]

**Comment by ra3orblade** [comment]
You need to sign CLA before I can merge.

**Comment by LavaCxx** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by fuksman** [comment]
@any contributor @LavaCxx code

**Comment by LavaCxx** [comment]
@ra3orblade  I have already signed the CLA.

**Pull request merged**